Neo4j.Driver/Neo4j.Driver/Internal/ConnectionPool.cs

```diff
@@ -111,6 +122,10 @@ public IPooledConnection CreateNewPooledConnection()
                 {
                     DestoryConnection(conn);
                 }
+                else
```

Comment:
Is this else required? Shouldn't we just decrement the _poolsize without looking at the conn's being null or not?